freckle-app/CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
-## [_Unreleased_](https://github.com/freckle/freckle-app/compare/freckle-app-v1.20.0.2...main)
+## [_Unreleased_](https://github.com/freckle/freckle-app/compare/freckle-app-v1.20.1.0...main)
 
-## [v1.20.0.2](https://github.com/freckle/freckle-app/compare/freckle-app-v1.20.0.1...freckle-app-v1.20.0.2)
```

Comment:
I apparently never actually made the version bump to release `1.20.0.2`, so those changes are going into `1.20.1.0` now.